app/models/ingest_job.rb

```diff
@@ -237,7 +237,7 @@ def set_study_state_after_ingest
     when 'Metadata'
       self.study.set_cell_count
       self.set_study_default_options
-      self.launch_subsample_jobs
+      #self.launch_subsample_jobs
```

Comment:
This looks like a merge issue when pulling from another branch - these should not be commented out.  I think Devon was doing this when we were trying to diagnose an issue yesterday with scp-ingest-pipeline.

Reply:
Addressed by updating with Devon's recent merge, b5db36c.